crates/nu-cli/tests/test_completions.rs

```diff
@@ -87,9 +86,26 @@ pub fn new_engine() -> (PathBuf, String, EngineState) {
 }
 
 // match a list of suggestions with the expected values
-pub fn match_suggestions(expected: Vec<String>, suggestions: Vec<Suggestion>) {
-    expected.iter().zip(suggestions).for_each(|it| {
-        assert_eq!(it.0, &it.1.value);
+// skipping the order (for now) due to some issue with sorting behaving
+// differently for each OS
+fn match_suggestions(expected: Vec<String>, suggestions: Vec<Suggestion>) {
```

Comment:
```suggestion
#[cfg(target_os = "windows")]
fn match_suggestions(expected: Vec<String>, suggestions: Vec<Suggestion>) {
    expected.iter().zip(suggestions).for_each(|it| {
        assert_eq!(it.0, &it.1.value);
    });
}

#[cfg(not(target_os = "windows"))]
fn match_suggestions(expected: Vec<String>, suggestions: Vec<Suggestion>) {
```
only test if not on windows; if on windows, use the old test which worked on there as well.